<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>any_zoom::ZoomDialog</class>
 <widget class="QDialog" name="any_zoom::ZoomDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>350</width>
    <height>107</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Zoom</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="x_label">
     <property name="text">
      <string>X zoom:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QSlider" name="x_zoom_slider">
     <property name="minimum">
      <number>9</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>425</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QSpinBox" name="x_zoom_edit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string notr="true">%</string>
     </property>
     <property name="minimum">
      <number>9</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>425</number>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="y_label">
     <property name="text">
      <string>Y zoom:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QSlider" name="y_zoom_slider">
     <property name="minimum">
      <number>9</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>425</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QSpinBox" name="y_zoom_edit">
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="suffix">
      <string notr="true">%</string>
     </property>
     <property name="minimum">
      <number>9</number>
     </property>
     <property name="maximum">
      <number>9999</number>
     </property>
     <property name="value">
      <number>425</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>any_zoom::ZoomDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>227</x>
     <y>87</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>any_zoom::ZoomDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>295</x>
     <y>93</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>106</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x_zoom_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>x_zoom_edit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>249</x>
     <y>19</y>
    </hint>
    <hint type="destinationlabel">
     <x>288</x>
     <y>16</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>x_zoom_edit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>x_zoom_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>275</x>
     <y>29</y>
    </hint>
    <hint type="destinationlabel">
     <x>250</x>
     <y>27</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_zoom_slider</sender>
   <signal>valueChanged(int)</signal>
   <receiver>y_zoom_edit</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>247</x>
     <y>50</y>
    </hint>
    <hint type="destinationlabel">
     <x>284</x>
     <y>47</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>y_zoom_edit</sender>
   <signal>valueChanged(int)</signal>
   <receiver>y_zoom_slider</receiver>
   <slot>setValue(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>278</x>
     <y>60</y>
    </hint>
    <hint type="destinationlabel">
     <x>256</x>
     <y>58</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
